tests/test_data.py

```diff
@@ -2,14 +2,20 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import os
+from pathlib import Path
 
+import access_nri_intake
 from access_nri_intake.utils import get_catalog_fp
```

Comment:
This breaks if you don't have the package installed at all; are we concerned about this? I think there's meant to be some settings that can be added to `pyproject.toml` to fix this, but I couldn't get them to work last time I tried...

Reply:
Hmmmm, good question. I'm guessing it probably doesn't matter - surely the following line will fail anyway if the package isn't installed at all.

In fact, I'm not sure that any sort of testing works if a package isn't installed - it's necessary that we import the functionality we're testing from somewhere, right?

Reply:
I vaguely remember having been able to do some testing on another package way back in the day without bothering with an install - I *think* there's some setting that sticks the local codebase into the top of `PYTHONPATH` when you invoke `pytest`. That being said, I've not been able to remind myself of what it was yet, so it's entirely possible I'm gibbering rubbish.

Reply:
Maybe [this](https://stackoverflow.com/a/54392877)? Pretty sure doing it this way would work too though?

Reply:
Ah yes, I had seen that one before. However, I thought there was a switch/setting you could throw at the top configuration level that had the same effect as `python -m`... might have a poke around this morning.

Reply:
I'm looking at a Stromlo project I've been working on ([AnniesLasso](https://github.com/ANU-RSAA/AnniesLasso)), and I can `pytest` that without having an install of the code. I can't dins anything in the setup of that repo that allows this, so I'm guessing the only reason I can do it is because of the repo structure (the tests are contains inside the main module code).